stable/selenium/templates/NOTES.txt

```diff
@@ -0,0 +1,89 @@
+Selenium Hub can be accessed via port {{ .Values.hub.port }} on the following 
+DNS name from within your cluster:
+
+- http://{{ template "hub.fullname" . }}.{{ .Release.Namespace }}:{{ .Values.hub.port }}
+
+You can easily connect to the remote instance from your browser. Forward the
+webserver port to localhost:4444
+
+- kubectl port-forward --namespace {{ .Release.Namespace }} \
+  $(kubectl get pods --namespace {{ .Release.Namespace }} \
+    -l app={{ template "hub.fullname" . }} \
+    -o jsonpath='{ .items[0].metadata.name }') 4444
+
+You can also connect to the container running Selenium Hub. To open a shell
+session in the pod run the following:
+
+- kubectl exec -i -t --namespace {{ .Release.Namespace }} \
+  $(kubectl get pods --namespace {{ .Release.Namespace }} \
+    -l app={{ template "hub.fullname" . }} \
+    -o jsonpath='{.items[0].metadata.name}') /bin/sh
+
+To tail the logs for the Selenium Hub pod run the following:
+
+- kubectl logs -f --namespace {{ .Release.Namespace }} \
+    $(kubectl get pods --namespace {{ .Release.Namespace }} \
+      -l app={{ template "hub.fullname" . }} \
+      -o jsonpath='{ .items[0].metadata.name }')
+
+{{- if eq .Values.hub.serviceType "LoadBalancer" }}
+
+To watch for the LoadBalancer IP run the following
+
+- kubectl get svc -w --namespace {{ .Release.Namespace }} \
+    -l app={{ template "hub.fullname" . }}
+{{- end }}
+
+{{- if and (not .Values.chrome.enabled) (not .Values.chromeDebug.enabled) (not .Values.firefox.enabled) (not .Values.firefoxDebug.enabled) }}
+WARNING: No nodes enabled, no browsers available via hub
+
+- Enable Chrome
+  helm upgrade --set chrome.enabled=true {{ .Release.Name }} selenium
```

Comment:
Ah, TIL you can use `{{ Template.BasePath }}` instead of hardcoding the chart path.